images/ami/flatcar.yaml

```
@@ -4,7 +4,7 @@ packer:
   # Selectors for source AMI:
   ami_filter_name: "Flatcar*stable*"
   ami_filter_owners: "075585003325"
-  source_ami: ""
+  source_ami: "ami-0e7e21b47b0e90b22" # Flatcar-3033.2.0 LTS
```

Maybe its failing the linter because of:
```suggestion
  # Flatcar-3033.2.0 LTS
  source_ami: "ami-0e7e21b47b0e90b22" 
```